### ISSUE TITLE
Integration 'crashes' projector 

### ISSUE THREAD

**Issue by Upke** [iloveicedgreentea/jvc_homeassistant#46]
Dear developer(s), 

Apparently this integration has the ability to lock up the projector. I've had this in two instances. The projector wouldn't respond to any commands, not from the remote control, nor from the buttons on the back (nor from home assistant). The only way to shut the projector down and regain control, is by cutting the power. Perhaps it's a bug, perhaps the way I'm using the integration. 

Details: 
Current home assistant version: 2024.4.4 (although I've had the same issue a couple of weeks ago on an older version, stil running on a RP4) 
Current integration version: pre-release 4.4.08
Projector: JVC NP5 (US: DLA-RS1100) 
System: NUC with intel N100 

My home cinema (with Denon receiver, JVC projector and Apple TV) is behind a Zigbee plug to cut down standby power usage. I've got an automation which stops and starts the integrations pertaining to the aforementioned devices). If I don't do it, my log's are filled with 'can't connect' errors.

Attached is my entire log file for the JVC integration but I'll quote some lines from it. Might be different issues.

On start up (and than repeat): 
_2024-04-24 13:29:44.960 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'00FD' is not a valid ResolutionModes
2024-04-24 13:29:44.960 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'00FD' is not a valid ResolutionModes
2024-04-24 13:29:45.764 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'IF0300PJ' is not a valid ColorSpaceModes
2024-04-24 13:29:45.765 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'IF0300PJ' is not a valid ColorSpaceModes
2024-04-24 13:29:46.168 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'IS0' is not a valid PowerModes
2024-04-24 13:29:46.169 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'IS0' is not a valid PowerModes
2024-04-24 13:29:46.570 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'PW1' is not a valid SourceStatuses
2024-04-24 13:29:46.570 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'PW1' is not a valid SourceStatuses
2024-04-24 13:29:46.972 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'SC1' is not a valid AnamorphicModes
2024-04-24 13:29:46.973 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'SC1' is not a valid AnamorphicModes
2024-04-24 13:29:47.374 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'IN0' is not a valid InputLevel
2024-04-24 13:29:47.374 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'IN0' is not a valid InputLevel
2024-04-24 13:29:47.775 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'3' is not a valid MaskModes
2024-04-24 13:29:47.775 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'3' is not a valid MaskModes
2024-04-24 13:29:48.176 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'IS2' is not a valid ContentTypes
2024-04-24 13:29:48.177 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'IS2' is not a valid ContentTypes
2024-04-24 13:29:48.979 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'PM0' is not a valid InputModes
2024-04-24 13:29:48.979 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'PM0' is not a valid InputModes
2024-04-24 13:29:49.382 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'IP6' is not a valid PictureModes
2024-04-24 13:29:49.382 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'IP6' is not a valid PictureModes
2024-04-24 13:29:53.358 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'PM03' is not a valid AspectRatioModes
2024-04-24 13:29:53.359 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'PM03' is not a valid AspectRatioModes
2024-04-24 13:29:53.761 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'IS3' is not a valid ContentTypeTrans
2024-04-24 13:29:53.762 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'IS3' is not a valid ContentTypeTrans
2024-04-24 13:29:54.162 ERROR (MainThread) [custom_components.jvc_projectors.remote] Attribute not found - b'PM1' is not a valid InstallationModes
2024-04-24 13:29:54.163 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: b'PM1' is not a valid InstallationModes
2024-04-24 13:29:54.564 ERROR (MainThread) [custom_components.jvc_projectors.remote] ValueError in handle_queue: invalid literal for int() with base 16: b'IN0'_

Then there is:
_2024-04-24 17:56:18.567 WARNING (MainThread) [asyncio] socket.send() raised exception. (x200 or so)_ 
 After which I noticed that the projector had locked up/crashed and was unresponsive. Cut the power en shut down the JVC add-on. Might be unrelated as I see errors in my scripts as well. 

Is there a way to research and troubleshoot this issue?

**Comment by Upke** [comment]
[home-assistant_jvc_projectors_2024-04-24T15-56-19.386Z.log](https://github.com/iloveicedgreentea/jvc_homeassistant/files/15099895/home-assistant_jvc_projectors_2024-04-24T15-56-19.386Z.log)

**Comment by iloveicedgreentea** [comment]
Damn I thought I fixed it. I will take a look

**Comment by iloveicedgreentea** [comment]
Just want to confirm you are definitely running the latest beta? Can you try to redownload and restart? I was able to reproduce this freeze in .07 and then can't reproduce it in .08.

The projector freezing is a JVC bug when it gets too many commands and their internal server crashes.

**Comment by Upke** [comment]
Thanks for your response! 
It says v4.4.08 on the hacs page but to be safe, I've redownloaded it and restarted home assistant. 
I'm a bit wary to use the integration for fear of another lockup. Not sure how much uncooled shutdowns my lamp can take.

**Comment by iloveicedgreentea** [comment]
Yeah thats understandable. If it happens again feel free to revert to the latest stable version

The previous lock up was caused by sending a command sometimes which I fixed and have not been able to replicate since. I will try to replicate in the meantime just in case

**Comment by iloveicedgreentea** [comment]
Verifying the version is crucial because I know for a fact it was a problem on 4.4.07 but I wasn't able to replicate it on 4.4.08 after adding some fixes

**Comment by Upke** [comment]
Hello, 

I've just had the projector lock up again. Not sure when it happend but noticed it when the image started showing artifacts and the projector wasn't able to switch between different modes (resolution and hz). I started the projector at about 14.10 and from 15.00 a whole line of errors:

2024-05-05 15:00:26.696 WARNING (MainThread) [asyncio] socket.send() raised exception.
At 16.02 I noticed the projector had locked up but could've happened a lot sooner. 
2024-05-05 16:02:41.893 WARNING (MainThread) [asyncio] socket.send() raised exception.

Running version v4.4.09.

You've set expectations about maintaining this integration quite clear in the other open issue. So this might get resolved, might not. No problem, I thank you for your effort!
